src/nvim/lua/converter.c

```diff
@@ -1241,8 +1241,18 @@ LuaRef nlua_pop_LuaRef(lua_State *const lstate, Error *err)
   type nlua_pop_##type(lua_State *lstate, Error *err) \
   FUNC_ATTR_NONNULL_ALL FUNC_ATTR_WARN_UNUSED_RESULT \
   { \
+    if (lua_type(lstate, -1) != LUA_TNUMBER) { \
+      lua_pop(lstate, 1); \
+      api_set_error(err, kErrorTypeValidation, "Expected lua number"); \
```

Comment:
Lua is a proper noun and should be capitalized:

```suggestion
      api_set_error(err, kErrorTypeValidation, "Expected Lua number"); \
```

This is inconsistent across our code base right now but I'm trying to pedantically correct it one code review at a time.